chall02/fulldemo.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/local/bin/python3
+
+import sys
```

Comment:
You should always protect the executable part of your code with
```python3
if __name__ == "__main__":
    # rest of your code
```
Or else this code will be executed even when importing things from your module.